content/en/docs/getting-started/first-project/_index.md

```diff
@@ -5,3 +5,62 @@ weight: 2
 description: >
   Now that you've setup the platform, let's create your first project.
 ---
+<style>
+.bg-lime {background: lime;}
+.bg-green {background: green;}
+.bg-black {background: black;}
+.bg-red {background: red;}
+</style>
+
+If you're just starting your development process with Jenkins X and want to start with some tested, supported project templates that you can setup quickly, you can create a **Quickstart**!
+
+Quickstarts are pre-made applications you can start a project from, instead of starting from scratch.
+
+
+<div class="row">
```

Comment:
Are these boxes really worth getting in so much HTML? IMO it really destroys the concept of using something like Hugo or AsciiDoctor.